.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] You are also supposed to ignore the node modules. This is because the node modules will takeup a lot of bandwidth to push tp GitHub because of its large size. as well as it is generally not good practice to include the node modules in your GitHub code base.

Reply:
@Nemwel-Boniface , thanks very much. 
.gitignore updated accordingly.